packages/drivers/odsp-driver/src/test/odspError.spec.ts

```diff
@@ -30,6 +31,26 @@ describe("Odsp Error", () => {
 		},
 	} as Response;
 
+	/**
+	 * Checks if the input is an {@link IGenericNetworkError}.
+	 */
+	function isIGenericNetworkError(input: any): input is IGenericNetworkError {
```

Comment:
I would recommend using `unknown` instead of `any` in these new functions. You'll also have to do something like the following in the code:
```typescript
(input as Partial<IGenericNetworkError>).errorType === ...
```

Reply:
As a general rule, we should almost never use `any`.